cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -1074,6 +1094,36 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
                 },
             };
         }
+        case AnnotationActionTypes.HIDE_SHOW_CONTEXT_IMAGE: {
+            const { hide } = action.payload;
+
+            return {
+                ...state,
+                player: {
+                    ...state.player,
+                    contextImage: {
+                        loaded: state.player.contextImage.loaded,
+                        data: state.player.contextImage.data,
```

Comment:
Use destruction please. To follow DRY principles
`const { loaded, data} = state.player.contextImage`